packages/react/src/components/button/index.tsx

```diff
@@ -72,7 +72,11 @@ const Button: React.FC<React.PropsWithChildren<ButtonProps>> = ({
   const contentProps = {rightIcon, leftIcon, children}
 
   return (
-    <button data-sr-button {...rest} disabled={isLoading || rest.disabled}>
+    <button
+      data-sr-button
+      {...rest}
+      disabled={isLoading || isDisabled || rest.disabled}
+    >
```

Comment:
This shared react package button was not taking the `isDisabled` prop/option into account, this adds it as one of the `disabled` conditions.

Reply:
wasn't it getting passed as part of `rest` props? I can see `rest.disabled` in there.